### PR TITLE
created message view activities

### PR DESCRIPTION
**Pull request by jacob-lu**
* worked on message view activities
* still need to work on keyboard again

### CODE REVIEW

File: app/src/main/java/org/colourhood/instamessage/CustomAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import android.view.ViewGroup
+import android.widget.TextView
+import org.w3c.dom.Text
+
+class CustomAdapter (val userList: ArrayList<User>): RecyclerView.Adapter<RecyclerView.ViewHolder>(){
+
+
+
+
+    override fun onCreateViewHolder(parent: ViewGroup?, viewType: Int): RecyclerView.ViewHolder {
```
Comment (jacob-lu) [inline app/src/main/java/org/colourhood/instamessage/CustomAdapter.kt]: hi

File: app/src/main/java/org/colourhood/instamessage/CustomAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import android.support.v7.widget.RecyclerView
+import android.text.Layout
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import org.w3c.dom.Text
+
+class CustomAdapter (val userList: ArrayList<User>): RecyclerView.Adapter<RecyclerView.ViewHolder>(){
+
```
Comment (katherineli0514) [inline app/src/main/java/org/colourhood/instamessage/CustomAdapter.kt]: Please delete the empty rows.

File: app/src/main/java/org/colourhood/instamessage/CustomAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            }
+        }
+        return ViewHolder1(view)
+    }
+
+    override fun getItemViewType(position: Int): Int {
+        return position %2 *2
+    }
+
+//    val v = LayoutInflater.from(parent?.context).inflate(R.layout.list_layout, parent, false)
```
Comment (katherineli0514) [inline app/src/main/java/org/colourhood/instamessage/CustomAdapter.kt]: Please delete the comment code.

File: app/src/main/java/org/colourhood/instamessage/MainActivity.kt [hunk 4]
```diff
@@ -2,11 +2,46 @@ package org.colourhood.instamessage
 
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
```
Comment (katherineli0514) [inline app/src/main/java/org/colourhood/instamessage/MainActivity.kt]: Please create a new activity named MessageViewActivity and copy the changes you did on MainActivity to the new activity. Please revoke all the changes you did on MainActivity.

File: app/src/main/java/org/colourhood/instamessage/MainActivity.kt [hunk 5]
```diff
@@ -2,11 +2,46 @@ package org.colourhood.instamessage
...
+import android.widget.EditText
+import android.widget.LinearLayout
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        val recyclerView = findViewById<RecyclerView>(R.id.recyclerView) as RecyclerView
```
Comment (katherineli0514) [inline app/src/main/java/org/colourhood/instamessage/MainActivity.kt]: I think you don't have to use as RecyclerView, as Button, as EditText.

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -1,18 +1,59 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (katherineli0514) [inline app/src/main/res/layout/activity_main.xml:2]: Samething for the MainActivity. Please create a new layout file called message_activity_view and do all the changes there.

**Review by katherineli0514 — APPROVED** [review]

**Pull request merged**